src/themes/components/tabs.ts

```diff
@@ -0,0 +1,41 @@
+const baseStyleTab = {
+  textStyle: 'body',
+  transitionProperty: 'common',
+  transitionDuration: 'normal',
+  borderBottom: '1.5px solid',
+  borderColor: 'transparent',
+  p: 'sm',
+  marginBottom: '-1.5px',
+  color: 'gray.600',
+  _selected: {
+    fontWeight: 'bold',
+    color: 'gray.900',
+    borderColor: 'currentColor',
+  },
+  _disabled: {
+    _active: { bg: 'none' },
+    cursor: 'not-allowed',
+    opacity: 0.4,
```

Comment:
@Marine-Berthier you need to reference the opacity through the object we already have in place instead hardcoding the opacity.